src/docs/DescriptionExplicit.stories.svelte

```diff
@@ -0,0 +1,51 @@
+<script lang="ts" context="module">
+	import { Story, Template } from '@storybook/addon-svelte-csf';
+	import type { Meta } from '@storybook/svelte';
+
+	import DescriptionComponent from './DescriptionComponent.svelte';
+
+	/**
+	 * This description was set above the `export const meta` inside the module tag `<script context="module">`.
+	 */
+	export const meta = {
+		title: 'Description Set Explicitly/Svelte CSF',
+		component: DescriptionComponent,
+		tags: ['autodocs'],
+		argTypes: {
+			useMeta: { table: { disable: true } },
+			useStory: { table: { disable: true } },
+		},
+		parameters: {
+			controls: { disable: true },
+			docs: {
+				description: {
+					component:
+						'This description was explicitly set in the `meta.parameters`. It will override the comment above the `export const meta`.',
+				},
+			},
+		},
+	} satisfies Meta<DescriptionComponent>;
+</script>
+
+<Template let:args>
+	<DescriptionComponent {...args} />
+</Template>
+
+<Story name="From export meta" args={{ useMeta: true }} tags={['!dev']} />
```

Comment:
I don't think this story is needed TBH, as it's already covered by "Not described"